File: backend/go/internal/models/session.go
package models

import "github.com/google/uuid"

type Session struct {
	ID        uuid.UUID `gorm:"primaryKey;type:uuid;default:uuid_generate_v4()" json:"id"`
	UserID    uint      `gorm:"column:user_id" json:"userId"`
	ExpiresAt int64     `gorm:"column:expires_at" json:"expiresAt"`
	CreatedAt int64     `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt int64     `gorm:"column:updated_at" json:"updatedAt"`

	User User `gorm:"foreignKey:UserID"`
}

File: backend/ASPNET/ASPNET/Models/Entities/Session.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ASPNET.Models.Entities
{
    public class Session
    {
        [Key]
        [StringLength(255)]
        public string Id { get; set; } = string.Empty;
        
        [Required]
        public int UserId { get; set; }
        
        [Required]
        public long ExpiresAt { get; set; }
        
        [Required]
        public long CreatedAt { get; set; }
        
        [Required]
        public long UpdatedAt { get; set; }

        // Navigation properties
        [ForeignKey("UserId")]
        public virtual User User { get; set; } = null!;
    }
}